GitUtils.ps1

```diff
@@ -115,12 +115,15 @@ function Get-GitStatus($gitDir = (Get-GitDirectory)) {
         $filesModified = @()
         $filesDeleted = @()
         $filesUnmerged = @()
+        $stashCount = 0
 
         if($settings.EnableFileStatus -and !$(InDisabledRepository)) {
             dbg 'Getting status' $sw
             $status = git -c color.status=false status --short --branch 2>$null
+            $stashCount = $null | git stash list 2>$null | measure-object | select -expand Count

```

Comment:
I tried `--no-pager` but unfortunately it didn't work. Good line of thought though, I didn't think about that. Is there anything else that could be causing git to grab all input like that?